tools/scripts/linux-bareflank-cross-compiler.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+export PREFIX="$HOME/opt/cross-compiler"
+export GCC_TARGET=x86_64-elf
+export BINUTILS_TARGET=x86_64-elf
+export PATH="$PREFIX/bin:$PATH"
+
+apt-get install build-essentials libmpc-dev libisl-dev
+
+mkdir -p $PREFIX
+
+pushd ~/Downloads
+#wget https://ftp.gnu.org/gnu/gcc/gcc-5.2.0/gcc-5.2.0.tar.bz2
```

Comment:
would it be possible to define these as variables that can be overridden if we want to try a different version? Would be useful for Travis CI